canister/src/runtime.rs

```diff
@@ -201,21 +201,25 @@ pub fn get_cycles_balance() -> u64 {
     CYCLES_BALANCE.with(|c| *c.borrow())
 }
 
-/// Returns the current time in seconds.
-#[cfg(target_arch = "wasm32")]
 pub fn time() -> u64 {
     // to get seconds from nanoseconds
-    ic_cdk::api::time() / 1_000_000_000
+    time_nanos() / 1_000_000_000
+}
+
+/// Returns the current time in seconds.
+#[cfg(target_arch = "wasm32")]
```

Comment:
unimportant for now, but just so that we're aware: since we now have wasm64, if there's ever the case that we compile this canister in 64-bit mode, we'll need to be careful here.

Reply:
noted, thank you!